### DISCUSSION TITLE
How does the cloud service work?

### DISCUSSION THREAD

**Discussion by joaopegoraro** [sequinstream/sequin#230] [General]
I see there's a cloud service I can quickly login with Github and start using. Is there more information about how does it work?

**Comment by thisisgoldman** [comment]
Hey @joaopegoraro - the cloud version of Sequin is identical to the self-hosted release. We just do the work to host it for you. Check out the "[How Sequin Works](https://sequinstream.com/docs/how-sequin-works)" section of our docs for more details - but please let us know if you have more specific questions.
